search-service/src/main/kotlin/com/egm/stellio/search/model/AttributeInstance.kt

```diff
@@ -10,5 +11,7 @@ data class AttributeInstance(
     val instanceId: URI = "urn:ngsi-ld:Instance:${UUID.randomUUID()}".toUri(),
     val observedAt: ZonedDateTime,
     val value: String? = null,
-    val measuredValue: Double? = null
+    val measuredValue: Double? = null,
+    // FIXME should not be null
+    val metadata: Json? = null
```

Comment:
- actually, there is not only metadata in this property, but the whole attribute payload
- and since you are migrating existing data, how could it be null?